examples/scripts/uptime.py

```diff
@@ -31,10 +31,10 @@ def v2_runner_on_failed(self, result, *args, **kwargs):
 
 def main():
     host_list = ['localhost', 'www.example.com', 'www.google.com']
-    Options = namedtuple('Options', ['connection', 'module_path', 'forks', 'remote_user',
-                                     'private_key_file', 'ssh_common_args', 'ssh_extra_args', 'sftp_extra_args',
-                                     'scp_extra_args', 'become', 'become_method', 'become_user', 'verbosity', 'check',
-                                     'diff'])
+    # since the API is constructed for CLI it expects certain options to always be set in the context object
+    context.CLIARGS = ImmutableDict(connection='local', module_path=['/usr/share/ansible'], forks=10, become=None,
```

Comment:
To match the previous behavior, we should have `connection='smart'` instead of `local`